ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -28,8 +28,7 @@ QtObject {
     property string signingPhrase: walletSection.signingPhrase
     property string mnemonicBackedUp: walletSection.isMnemonicBackedUp
 
-    property var collections: walletSectionCollectibles.model
-    property var flatCollectibles: walletSectionCollectibles.flatModel
```

Comment:
How about you preserve the `flatCollectibles` name? Then we (I) don't have to change existing code; afaics the new model didn't remove any roleNames, only added some new, so it should be backwards compatible? ;)

Reply:
done